bool success
#string current_order_id
# string order_kitting_shipment_type
# string order_kitting_agv_id
# string order_kitting_destination_id
# string part_type
# string order_assembly_station_id
# string order_assembly_shipment_type
Order order